msal/src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -260,10 +258,16 @@ private void VerifyAuthorizationResult()
                     MsalErrorMessage.NoPromptFailedErrorMessage);
             }
 
+            if (_authorizationResult.Status == AuthorizationStatus.UserCancel)
+            {
+                throw new MsalClientException(_authorizationResult.Error,
+                _authorizationResult.ErrorDescription);
```

Comment:
>_authorizationResult.ErrorDescription [](start = 16, length = 37)

also thinking the message should be specialized like the error code.

Reply:
sounds good.